src/main/scala/com/bazaarvoice/sswf/SSWFStep.scala

```diff
@@ -0,0 +1,19 @@
+package com.bazaarvoice.sswf
+
+/**
+ * Your Step Enum must implement/extend this
+ */
+trait SSWFStep {
+  /** The name of the Step */
+  def name: String
+  /** The version of the step definition (increment if you change any options) */
```

Comment:
It may be a good idea to expound on this statement in the documentation. Until you start using it, it's not clear that the version is what you'd want to use to control anyone that's currently doing work but not compatible with existing workflows. If I'm familiar with SWF, I might choose a different domain entirely or something to achieve this effect. Maybe a section called "patterns" that gives some idea of how SSWF could be used or something. Do you think it's clear enough already?